packages/mdc-tab-scroller/mdc-tab-scroller.scss

```diff
@@ -0,0 +1,50 @@
+/**
+  * @license
+  * Copyright 2018 Google Inc. All Rights Reserved.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License")
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+*/
+
+@import "@material/animation/variables";
+
+.mdc-tab-scroller {
+  overflow-y: hidden;
+}
+
+.mdc-tab-scroller__scroll-area {
+  -webkit-overflow-scrolling: touch;
+  display: flex;
+  margin-bottom: -18px;
+  overflow-x: scroll;
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
+}
+
+.mdc-tab-scroller__scroll-area--animating {
+  -webkit-overflow-scrolling: auto;
+}
+
+.mdc-tab-scroller__scroll-content {
+  display: flex;
+  flex: 1 0 auto;
+  padding-bottom: 18px;
+  transform: none;
+  transform-origin: left;
+  will-change: transform;
+}
+
+.mdc-tab-scroller__scroll-area--animating .mdc-tab-scroller__scroll-content {
```

Comment:
then this would be `.mdc-tab-scroller--animating .mdc-tab-scroller__scroll-content` which is self-evident that it is passing BEM. The way it is written right now you're not sure that `mdc-tab-scroller__scroll-area` is a parent to `mdc-tab-scroller__scroll-content` unless you double check the HTML README

Reply:
Sounds good.